transports/tls/Cargo.toml

```diff
@@ -16,7 +16,7 @@ libp2p-identity = { workspace = true }
 rcgen = "0.10.0"
 ring = "0.16.20"
 thiserror = "1.0.44"
-webpki = { version = "0.22.0", features = ["std"] }
+webpki = { version = "0.101.4", package = "rustls-webpki", features = ["std"] }
```

Comment:
This needs a version bump of `libp2p-tls`. In case we don't expose the `webpki` dependency on the public interface, we can release this as a patch release, namely `v0.2.1`. I still need to confirm the assumption that it is not exposed.

Reply:
Done!